app/components/post-search-results.js

```diff
@@ -148,19 +148,22 @@ export default Component.extend({
       });
   },
 
-  update(state) {
+  update(state, query = '') {
```

Comment:
Is the `query` used in the function?

Reply:
https://github.com/Wikia/mobile-wiki/pull/1808/commits/012427775fd963d54ad2e11aa0f49b61977945d8